vmngclient/primitives/primitive_container.py

```diff
@@ -17,3 +18,4 @@ def __init__(self, session: vManageSession):
         self.tenant_management = TenantManagementPrimitives(session)
         self.tenant_backup_restore = TenantBackupRestorePrimitives(session)
         self.tenant_migration = TenantMigrationPrimitives(session)
+        self.configuration_dashboard_status_primitives = ConfigurationDashboardStatusPrimitives(session)
```

Comment:
variable name without *_primitive suffix to keep it short and consistent with others